### ISSUE TITLE
How to remove unwanted warning while using GPU with tensorflow

### ISSUE THREAD

**Issue by prithagupta** [tensorflow/tensorflow#15720]
### System information
- I am using tensorflow version :
- OS Platform and Distribution: Linux64-:
- TensorFlow installed from pip version 1.4.1
- Python 3.6.3 :: Anaconda custom (64-bit) 
- Cuda-8.0
- GPU model and memory: nVIDIA K20 (Kepler)

### Describe the problem
I am using Adam Optimizer with a normal sequential network created via keras using tensorflow as backend. 

I get the following logs repeatedly for fitting, and creating the network. I also applied batch-normalization for the Dense Layer

/job:localhost/replica:0/task:0/device:GPU:0 -> device: 0, name: Tesla K20Xm, pci bus id: 0000:84:00.0, compute capability: 3.5
Adam_2/decay: (VariableV2): /job:localhost/replica:0/task:0/device:GPU:0
IsVariableInitialized_14: (IsVariableInitialized): /job:localhost/replica:0/task:0/device:GPU:0
Adam_2/decay/read: (Identity): /job:localhost/replica:0/task:0/device:GPU:0
Adam_2/decay/Assign: (Assign): /job:localhost/replica:0/task:0/device:GPU:0
Adam_2/beta_2: (VariableV2): /job:localhost/replica:0/task:0/device:GPU:0
IsVariableInitialized_13: (IsVariableInitialized): /job:localhost/replica:0/task:0/device:GPU:0

training/Adam/gradients/batch_normalization_3_1/batchnorm/mul_grad/Shape_1: (Const): /job:localhost/replica:0/task:0/device:GPU:0
training/Adam/gradients/batch_normalization_3_1/batchnorm/mul_grad/Shape: (Const): /job:localhost/replica:0/task:0/device:GPU:0
training/Adam/gradients/batch_normalization_3_1/moments/Squeeze_grad/Shape: (Const): /job:localhost/replica:0/task:0/device:GPU:0
training/Adam/gradients/zeros_87/Const: (Const): /job:localhost/replica:0/task:0/device:GPU:0

How to turn off these unwanted logs? 
I have already applied the following solution for switching off the warning logs from the tensorflow.
https://stackoverflow.com/questions/35911252/disable-tensorflow-debugging-information

### Source code 
from keras.layers.normalization import BatchNormalization

model = Sequential()

model.add(Dense(64, input_dim=14, init='relu'))
model.add(BatchNormalization())
model.add(Activation('relu'))

model.add(Dense(64, init='uniform'))
model.add(BatchNormalization())
model.add(Activation('relu'))

model.add(Dense(1, init='uniform'))
model.add(BatchNormalization())
model.add(Activation('softmax'))

model.compile(loss='binary_crossentropy', optimizer=Adam())

model.fit(X_train, y_train)

**Comment by skye** [comment]
@vrv can you take a look or redirect to someone who can? Thanks.

**Comment by vrv** [comment]
Added XQ who can redirect to someone who knows how the GPU info logging should work.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
A member of the TensorFlow organization has replied after the stat:awaiting tensorflower label was applied.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @zheng-xq: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @zheng-xq: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @zheng-xq: It has been 143 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @zheng-xq: It has been 158 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @zheng-xq: It has been 173 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @zheng-xq: It has been 188 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @zheng-xq: It has been 203 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @zheng-xq: It has been 218 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @zheng-xq: It has been 233 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @zheng-xq: It has been 248 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tfboyd: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tfboyd: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tfboyd: It has been 44 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tfboyd: It has been 59 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tfboyd: It has been 74 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by JoanaGodinho** [comment]
Has this been solved?
I have the same issue, I've tried doing 
os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3' 
but i keep having the prints. My OS is Windows10 and i'm using Cuda-9.0

**Comment by ymodak** [comment]
Closing due to staleness. Please check with the latest version of TensorFlow. Feel free to reopen if the issue still persists. Thanks!
